index.html

```diff
@@ -155,81 +142,65 @@ <h3 id="the-difficulties-of-ruby-processing">The Difficulties of Ruby Processing
 in order to decide on the position:</p>
 
 <ol>
-  <li id="l20200529001">How to handle the correspondence between the base characters and the ruby</li>
-  <li id="l20200529002">What to do when the string of base characters is longer than the ruby string</li>
-  <li id="l20200529003">What to do when the string of base characters is shorter than the ruby string</li>
-  <li id="l20200529004">
-    <p>When the ruby string protrudes from the base character string,
-    whether it can be allowed to be laid over the characters preceding or following,
-    and whether this affects the position of the base characters</p>
-    <aside class=note title="Protrusion of ruby from base characters" id="n20200529002">
-      From principle not to have extra spacing between characters in Japanese composition, 
-      one method of layout is to lay over the ruby string preceding or 
-      following 
-      <a href="https://www.w3.org/TR/jlreq/#cl-15">Hiragana (cl-15)</a> [[JLREQ]] 
-      characters but not to lay over the ruby string preceding 
-      or following Kanji characters. 
+  <li id="l20200529001">How to handle the correspondence between the base text and the ruby annotation.</li>
+  <li id="l20200529002">What to do when the  base text is wider than its ruby annotation.</li>
+<li id="l20200529003">What to do when the ruby annotation   is  wider than its base text.</li>
+<li id="l20200529004">
+    <p>When the ruby annotation is wider than its base text,
+    whether it can be allowed to overhang the characters preceding or following,
+    and whether this affects the position of the base text.</p>
+	
+	
+<aside class="note" title="Ruby annotations that are wider than the base text" id="n20200529002">
+<p>Since Japanese composition seeks to avoid space between characters as much as possible, ruby annotations wider than their base characters are not allowed to overhang neighboring kanji, but often <em>are</em> allowed to overhang neighboring kana. 
```

Comment:
we'd use 'spacing' rather than 'space', when it means to have extra space between characters but not to insert space character. So propose to change from "seeks to avoid space between characters" to "seeks to avoid spacing between characters".

Reply:
There are fine points that could be made for either, but since they are so fine i made the requested edit.